.github/workflows/openvex.yaml

```diff
@@ -0,0 +1,17 @@
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - 'v*.*.*'
+jobs:
+  vexctl:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set env
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - uses: openvex/generate-vex@31b415924ea0d72ed5f2640f1dee59dea6c2770b
+        name: Run vexctl
+        with:
+            product: pkg:golang/sigs.k8s.io/kueue@${{ env.RELEASE_VERSION }}
```

Comment:
@puerco can you verify that the purl here is correct?

Reply:
the previous step set's the envVar 
```yaml
      - name: Set env
        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
```

Reply:
So it is OK

Reply:
@psschwei Hey, How did you derive at this purl?

Reply:
from the purl-spec repo: https://github.com/package-url/purl-spec

Reply:
Thanks!